### DISCUSSION TITLE
Docker, Hasura and Azure SQL Edge

### DISCUSSION THREAD

**Discussion by ghost** [hasura/graphql-engine#9464] [Question]
I have a database hosted in a local docker container of Azure SQL Edge.  I am able to access it with Visual Studio Code, Azure Data Studio and the JavaScript MS SQL driver in a JS app.

I have hasura/graphql-engine community edition running is a separate docker container.

I am unable to get the Hasura Console to connect to the database.  I am not sure if it is an error in the connection string or something to do with hasura and SQL Edge in separate containers.

Any help appreciated!

**Comment by SamirTalwar** [comment]
Hey @dlovingst, can you please share your Docker configuration? Do you have a Docker Compose file you're using? If not, can you share the command-line invocations you're using to start everything up, including the arguments you're passing?

**Reply by ghost** [reply]
I have a hasura docker-compose.yml that spins up hasura and postgres so the hasura engine will run.  (attached)
[docker-compose.yml.zip](https://github.com/hasura/graphql-engine/files/10815348/docker-compose.yml.zip)
